portal/templates/profile/profile_macros.html

```diff
@@ -859,89 +859,108 @@ <h4 class="text-muted">{% if person and person.username %}{{ _("for") + " " + pe
-                    <ul class="list">
-                        <li v-for="item in subStudyTriggers.domains" v-text="item" class="text"></li>
-                    </ul>
+<!-- require a static bookmark link for post intervention questionnaire section for the dynamically generated content. Othewise the hash location won't go to the section correctly -->
+<a href="#postInterventionQuestionnaireLoc"></a>
+<div id="postInterventionQuestionnaireLoc">&nbsp;</div>
+<div class="row" id="postInterventionQuestionnaireSection" class="eproms-substudy" v-show="shouldShowPostTxQuestionnaireSection()"> 
+    <div class="col-md-12 col-xs-12">
+        {% call profileSection(editable=false) %}
+        {% call titleSection() -%}{{_("IRONMAN EMPRO")}}{%- endcall %}
+            <div id="postTxQuestionnaireContainer" class="eproms-substudy" data-profile-section-id="postTxQuestionnaire">
+                <div class="loader" v-show="postTxQuestionnaire.loading"><i class="fa fa-spinner fa-spin fa-2x icon"></i></div>
+                <!-- display trigger areas if indicated -->
+                <div id="triggersSection" v-if="hasSubStudyTriggers()">
+                    <h5 class="muted" v-show="!isSubStudyTriggersResolved()">{{_("Actions Required (for Clinicians/Site Staff)")}}</h5>
+                    <div class="item">{{_("EMPRO questionnaire completed on:")}} <span class="text" v-text="subStudyTriggers.date"></span></div>
+                    <div class="item">{{_("ACTION REQUIRED for high distress areas:")}}
+                        <ul class="list">
+                            <li v-for="item in subStudyTriggers.domains" v-text="item" class="text"></li>
+                        </ul>
+                    </div>
                 </div>
-            </div>
-            <div id="reportNoteSection">{{_("View subject EMPRO report for more information")}}</div>
-            <div id="questionsSection" class="content" :class="{disabled: isSubStudyTriggersResolved()}">
-                <div class="question" v-for="item in postTxQuestionnaire.questions" :linkId="item.linkId" :dataType="item.type" :text="item.text" >
-                    <!-- question text -->
-                    <span class="title" :class="item.type" v-text="item.text"></span>
-                    <!-- date type answer-->
-                    <input type="text" class="data-datepicker form-control answer" readonly placeholder="d M yyyy"
-                            :class="{disabled: isSubStudyTriggersResolved()}"
-                            :dataType="item.type"
+                <!-- no triggers indicated -->
+                <!-- show the last completed EMPRO date && indicate no triggers -->
+                <div id="noTriggersSection" v-else>
+                    <div class="item">{{_("Last EMPRO questionnaire completed on:")}} <span class="text" v-text="getLastSubStudyAssessmentDate()"></span></div>
+                    <div class="item text-muted">{{_("No high distress areas indicated")}}</div>
```

Comment:
Should this technically need a hyphen? :nerd_face: 
```suggestion
                    <div class="item text-muted">{{_("No high-distress areas indicated")}}</div>
```

Reply:
Reasonable, let's leave for now and let QA address.